### PR TITLE
php@7.2 7.2.34

### PR DESCRIPTION
**Pull request by chapeupreto**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Reported at https://bugs.php.net/bug.php?id=80171

File: Formula/php@7.2.rb [hunk 1]
```diff
@@ -2,9 +2,8 @@ class PhpAT72 < Formula
   desc "General-purpose scripting language"
   homepage "https://www.php.net/"
   # Should only be updated if the new version is announced on the homepage, https://www.php.net/
-  url "https://www.php.net/distributions/php-7.2.33.tar.xz"
-  mirror "https://fossies.org/linux/www/php-7.2.33.tar.xz"
```
Comment (derrabus) [inline Formula/php@7.2.rb:6]: We should keep the mirror.

Comment (chapeupreto) [reply Formula/php@7.2.rb:6]: You're right. I have no idea why that got removed. Fixed now. Thanks!

**Comment by fxcoudert** [comment]
```
/private/tmp/php@7.2-20201004-16362-2rofrq/php-7.2.34/main/reentrancy.c:139:23: error: too few arguments to function call, expected 3, have 2
        readdir_r(dirp, entry);
        ~~~~~~~~~            ^
/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/dirent.h:110:1: note: 'readdir_r' declared here
int readdir_r(DIR *, struct dirent *, struct dirent **) __DARWIN_INODE64(readdir_r);
```

**Review by bayandin — APPROVED** [review]

**Pull request closed without merging**